DEVELOPMENT.md

```diff
@@ -0,0 +1,50 @@
+# Notes for developers
+
+## Building
+
+You need Go version 1.14 to build this project.
```

Comment:
Do we need 1.14 ?

I note that `go.mod` indicates that it requires 1.13?

Reply:
I've checked and go v1.13 is fine; I've updated the text.